Doc/whatsnew/3.8.rst

```diff
@@ -163,6 +163,14 @@ gettext
 Added :func:`~gettext.pgettext` and its variants.
 (Contributed by Franz Glasner, Éric Araujo, and Cheryl Sabella in :issue:`2504`.)
 
+
+gc
+--
```

Comment:
I'm not sure that it's rendered correctly without an empty line between "---" and the new paragraph.

Reply:
Done in 5b62014